contact.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+
+    <meta charset="utf-8">
+
+    <title>Sprint Challenge - Contact</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+
+
+</head>
+
+<body>
+
+    <nav class="top-nav">
+        <img class="nav-logo" src="/img/lambda-black.png" alt="lambda navigation logo">
+        <a class="nav-link" href="/index.html">Home</a>
+        <a class="nav-link" href="/about.html">About</a>
+        <a class="nav-link" href="#">Products</a>
+        <a class="nav-link" href="#">Blog</a>
+        <a class="nav-link" href="/contact.html">Contact</a>
+
+    </nav>
+
+    <header>
+        <img src="/img/contact-us-img.jpg" alt="header about page image">
+    </header>
+
+    <div class="contact-us-container">
+        <h1>Contact Us</h1>
+        <form class="contact-form">
+            <input class="text-input" type="text" placeholder="Your first name...">
+            
+            <input class="text-input" type="text" placeholder="Your last name...">
+
+            <textarea name="message" id="message-area" cols="30" rows="10" placeholder="Your message..."></textarea>
+
+            <input class="submit-btn" type="button" value="Submit">
+        </form>
+    </div>
+
+
+    <!-- FOOTER SECTION -->
+    <footer class="about-page-footer">
+            <nav class="footer-nav">
+              <a href="index.html">Home</a>
+              <a href="/about.html">About</a>
+              <a href="#">Products</a>
+              <a href="#">Blog</a>
+              <a href="#">Contact</a>
+            </nav>
+          </footer>
+
+</body>
+
+</html>
```

Comment:
https://github.com/MicahJank/Sprint-Challenge--User-Interface/blob/f6044213aaacafacdb293031a2ef32ea8a1706f9/contact.html#L1-L60

great work on this section, adding it to the overall application, and including the form inputs and styling them.